_posts/2024-09-30-Boosting-OpenSearch-Development-on-MacOS-with-Apple-Silicon.md

```diff
@@ -0,0 +1,234 @@
+meta_description: Learn how to simplify your OpenSearch development workflow on macOS with Apple silicon
+---
+
+OpenSearch is an Apache-2.0 licensed open-source project supported by an engaging community of users across the globe. Building on its commitment to openness and growth, OpenSearch recently became part of the newly formed [OpenSearch Software Foundation](https://foundation.opensearch.org/), a community-driven initiative under the [Linux Foundation](https://www.linuxfoundation.org/). This shift has allowed us to enhance the developer experience by expanding platform and architecture support, especially for developers using macOS ARM64 machines. Previously, setting up OpenSearch cluster and running integration tests required a Linux environment or Docker containers, adding complexity and overhead to the process. Now, you can run and test OpenSearch plugins natively on macOS, simplifying workflows and reducing development time.
+
+We're excited to announce that the OpenSearch core snapshot tarball artifact, built specifically for macOS ARM64 machines, is now publicly available. If you're developing on these machines, especially the latest Mac computers with Apple silicon, we encourage you to give it a try.
+
+In this blog post, we'll discuss accessing the new snapshot artifact and using it to streamline your OpenSearch and OpenSearch Dashboards development workflow.
+
+
+## Prerequisite
+
+You can use the artifact described in this post on a [Mac computer with Apple silicon](https://support.apple.com/en-us/116943) (ARM64 architecture). For older models with the Intel processor, you can use the [OpenSearch core snapshot artifact for macOS X64 Host](https://github.com/opensearch-project/opensearch-build/issues/2216).
+
+
+## Download the artifact and start an OpenSearch cluster
+
+You can download the artifact tarball through one of the following links:
+
+- [1.3.19 snapshot artifact](https://artifacts.opensearch.org/snapshots/core/opensearch/1.3.19-SNAPSHOT/opensearch-min-1.3.19-SNAPSHOT-darwin-arm64-latest.tar.gz)
+
+- [2.16.0 snapshot artifact](https://artifacts.opensearch.org/snapshots/core/opensearch/2.16.0-SNAPSHOT/opensearch-min-2.16.0-SNAPSHOT-darwin-arm64-latest.tar.gz)
+
+- [3.0.0 snapshot artifact](https://artifacts.opensearch.org/snapshots/core/opensearch/3.0.0-SNAPSHOT/opensearch-min-3.0.0-SNAPSHOT-darwin-arm64-latest.tar.gz)
+
+Our pipeline is configured to publish new artifact versions whenever they become available.
+
+To start the cluster, run the following commands:
+
+```bash
+# Download 3.0.0 core snapshot
+wget https://artifacts.opensearch.org/snapshots/core/opensearch/3.0.0-SNAPSHOT/opensearch-min-3.0.0-SNAPSHOT-darwin-arm64-latest.tar.gz
+
+# Extract the tarball
+tar -xzvf opensearch-min-3.0.0-SNAPSHOT-darwin-arm64-latest.tar.gz
+
+# Start the cluster
+cd opensearch-3.0.0-SNAPSHOT
+./bin/opensearch
+```
+
+Once the cluster is started, you can make a test API call to verify that the cluster is working:
```

Comment:
"make" => "send"?

Reply:
You make a call but you send a request 😄 so leaving as is.